docs/debugging.md

```diff
@@ -316,6 +316,21 @@ PYTHON_GIL=0 python -c "import sys; print(sys._is_gil_enabled())"
 cd ..
 ```
 
+!!! note
+    On MacOS, you may see messages like this when you start Python:
+
+    ```
+    python(7027,0x1f6dfc240) malloc: nano zone abandoned due to inability to reserve vm space.
+    ```
+
+    This message is being emmitted by the MacOS malloc implementation. As
+    [explained
+    here](https://stackoverflow.com/questions/64126942/malloc-nano-zone-abandoned-due-to-inability-to-preallocate-reserved-vm-space),
+    this happens for any program compiled with thread sanitizer on MacOS and can
+    be safely ignored by setting the `MallocNanoZone` environment variable to
+    0\. You should only set this in session you are running thread sanitizer
```

Comment:
a typo snuck in here:

```suggestion
    0. You should only set this in session you are running thread sanitizer
```

Reply:
Apparently not. Looks like a bug in the pre-commit thingy, but whatever - it renders the same